test/unit/feirari/AutoRewardsDistributor.test.ts

```diff
@@ -82,8 +67,7 @@ describe('AutoRewardsDistributor', function () {
 
   describe('Init', function () {
```

Comment:
This init tests a bunch of stuff that isn't the AutoRewardsDistributor. Init should test the state of the contract being tested only

Reply:
fixed.